management-app/src/components/CreateModal/CreateModal.tsx

```diff
@@ -84,7 +124,7 @@ function CreateModal({
           {titleModal}
         </Typography>
 
-        <Box component="form" onSubmit={onCreate} sx={{ Width: '600px' }}>
+        <Box component="form" onSubmit={handelSubmit} sx={{ Width: '600px' }}>
```

Comment:
sx={{ Width: '600px' }} 
Width is ignored in this case because of the capital "W"

Reply:
and for PIXELS you can omit quotation mark and 'px' like this:  sx={{ width: 600 }}

Reply:
This code is  unnecessary, I deleted it. Thanks for our attention.